corpus/convert.py

```diff
@@ -326,3 +330,69 @@ def run(self):
                     segment.fullname() in segments_whitelist
                 ):
                     f.write(segment.orth + "\n")
+
+
+class CorpusToRawWavHDFJob(Job):
+    """
+    Gets audio files from a Bliss corpus and stores them as HDF file
+    compatible with the returnn HDFDataset
+    """
+
+    def __init__(
+        self,
+        bliss_corpus: tk.Path,
+        segment_file: Optional[tk.Path] = None,
+        returnn_root: Optional[tk.Path] = None,
+    ):
+        """
+
+        :param bliss_corpus: Bliss corpus to read segments and audio files from
+        :param segment_file: segment file that lists allowed segments
+        :param returnn_root: returnn repository
+        """
+        self.set_vis_name("Dump audio to HDF")
+
+        self.bliss_corpus = bliss_corpus
+        self.segment_file = segment_file
+        self.returnn_root = returnn_root
+
+        self.out_hdf = self.output_path("audio.hdf")
+
+    def tasks(self):
+        yield Task("run", mini_task=True)
+
+    def run(self):
+        SimpleHDFWriter = get_returnn_simple_hdf_writer(self.returnn_root.get_path())
```

Comment:
This doesn't seem to work if `returnn_root` is `None`.

Reply:
you are right.. the main problem is that the function `get_returnn_simple_hdf_writer` wants a str instead of a path. Maybe we want to change that?

otherwise:
```suggestion
        returnn_root = None if self.returnn_root is None else self.returnn_root.get_path()
        SimpleHDFWriter = get_returnn_simple_hdf_writer(returnn_root)
```

Reply:
> Maybe we want to change that?

I used str because it is an internal function...